pkg/performance/capabilities/capabilities_test.go

```diff
@@ -0,0 +1,110 @@
+// Copyright Antimetal, Inc. All rights reserved.
+//
+// Use of this source code is governed by a source available license that can be found in the
+// LICENSE file or at:
+// https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt
+
+package capabilities
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCapability_String(t *testing.T) {
+	tests := []struct {
+		name string
+		cap  Capability
+		want string
+	}{
+		{
+			name: "CAP_SYS_ADMIN",
+			cap:  CAP_SYS_ADMIN,
+			want: "CAP_SYS_ADMIN",
+		},
+		{
+			name: "CAP_SYSLOG",
+			cap:  CAP_SYSLOG,
+			want: "CAP_SYSLOG",
+		},
+		{
+			name: "CAP_BPF",
+			cap:  CAP_BPF,
+			want: "CAP_BPF",
+		},
+		{
+			name: "CAP_PERFMON",
+			cap:  CAP_PERFMON,
+			want: "CAP_PERFMON",
+		},
+		{
+			name: "Unknown capability",
+			cap:  Capability(999),
+			want: "UNKNOWN",
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			assert.Equal(t, tt.want, tt.cap.String())
+		})
+	}
+}
+
+func TestGetEBPFCapabilities(t *testing.T) {
```

Comment:
Same here. Any change to the ebpf capabilities requires updating this test. So why have this test?